phost/src/chain_client.rs

```diff
@@ -175,6 +175,22 @@ pub fn storage_value_key_vec(module: &str, storage_key_name: &str) -> Vec<u8> {
     key
 }
 
+fn storage_map_key_vec(module: &str, storage_item: &str, storage_item_key: &str) -> Vec<u8> {
+	let mut key = storage_value_key_vec(module, storage_item);
+	let item_key = hex::decode(storage_item_key).unwrap();
+	let hash = twox_64(&item_key);
+	key.extend(&hash);
+	key.extend(&item_key);
+	key
+}
+
+pub async fn get_parachain_heads(client: &XtClient, paraclient: &XtClient, hash: Option<Hash>) -> Option<Vec<u8>> {
```

Comment:
Let's move it after `pub fn check_round_end_event`

Reply:
Done